docs/tools/code-extension.mdx

```diff
@@ -18,6 +18,22 @@ src/
 src-gen/Foo
 ```
 
+Overall, we encourage you to open your project in VS Code using the following structure to fully leverage the features of the VS Code extension:
```

Comment:
```suggestion
:::note
We recommend selecting the root of your Lingua Franca package as your workspace in VS Code. This will enable the package explorer.
:::
```